### PR TITLE
Evaluate the removal of DCType

### PR DESCRIPTION
**Pull request by greg0ire**
Just trying to figure out the impact of not having such comments.

### CODE REVIEW

**Comment by SenseException** [comment]
```
  [Composer\Downloader\TransportException]   
  Could not authenticate against github.com  
```

The breaking workflows are not related to failing tests.

**Comment by greg0ire** [comment]
Yes; I need to figure out how to use a vcs repo in the CI

File: composer.json [hunk 1]
```diff
@@ -66,5 +66,11 @@
     "bin": ["bin/doctrine"],
     "archive": {
         "exclude": ["!vendor", "tests", "*phpunit.xml", "build.xml", "build.properties", "composer.phar", "vendor/satooshi", "lib/vendor", "*.swp"]
+    },
+    "repositories": {
+        "greg0ire": {
+            "type": "vcs",
+            "url": "https://github.com/greg0ire/dbal"
```
Comment (derrabus) [inline composer.json]:
```suggestion
            "type": "git",
            "url": "git@github.com:greg0ire/dbal.git"
```

Comment (greg0ire) [reply composer.json]: Oh right, obviously! Thanks @derrabus !

Comment (greg0ire) [reply composer.json]: Or maybe that wasn't the correct way after all…

**Comment by greg0ire** [comment]
Well, I guess you will just have to believe me when I say tests pass on both the ORM and the migrations repository :stuck_out_tongue:

**Pull request closed without merging**